### DISCUSSION TITLE
Docker文件权限问题？[Issue with Docker Linux File Permission]

### DISCUSSION THREAD

**Discussion by aYenx** [KrisCris/Palworld-Pal-Editor#51] [Q&A] [answered]
即便支持了docker，该项目依旧不支持Linux服务器存档的编辑吗？
![image](https://github.com/user-attachments/assets/93cc8307-fd5d-425d-aded-cc0c61f06047)

**Comment by KrisCris** [comment]
you will have to provide the correct gid & uid

**Reply by KrisCris** [reply]
I mean the UID and GID of your system. Unfortunately, you won't be able to access the file until you fix the permission issue.

![image](https://github.com/user-attachments/assets/e40541c7-c5e3-427d-8099-e6a93b69e2d4)

You would either let both docker containers run on the same UID and GID, or set the permission of Palworld files to something like 766

**Reply by aYenx** [reply]
是的，感谢你的回答，我刚刚发现是docker的PUID和PGID的设置问题，我正在尝试解决

**Reply by KrisCris** [reply]
I have no issue with saving. Can you provide the log

**Reply by aYenx** [reply]
你是指这个日志吗？
![image](https://github.com/user-attachments/assets/dcf0a82c-e794-4e8f-927b-5b31ceae8208)

**Reply by aYenx** [reply, marked as answer]
最终的解决方法：
1.将docker中的PUID&PGID都设置为0（我曾尝试设置为普通用户的UID，但好像出错了，大家可以自行尝试）
2.停止服务器运行并将sav文件全部改为766的权限(chmod 766 *.sav）
